packages/orchestration/src/vat-orchestration.js

```diff
@@ -1,13 +1,16 @@
 import { Far } from '@endo/far';
+import { prepareVowTools } from '@agoric/vow/vat.js';
 import { makeDurableZone } from '@agoric/zone/durable.js';
 import { prepareOrchestrationTools } from './service.js';
 
 /** @import {OrchestrationPowers} from './service.js' */
 
 export const buildRootObject = (_vatPowers, _args, baggage) => {
   const zone = makeDurableZone(baggage);
+  const vowTools = prepareVowTools(zone.subZone('VowTools'));
```

Comment:
is the subzone necessary?

Reply:
I'm not sure. I was following the example here:
https://github.com/Agoric/agoric-sdk/blob/1b838d258ed6e0baa2ec2476250c69a7719d271a/packages/vow/README.md?plain=1#L57-L77

Reply:
At least for the network and ibc vats, we've followed the pattern of creating a subzone specifically for all vow-related data

Reply:
Why? What happens if you don't make a subzone?